lpstat: Add page (#9119)
tldr-pages/tldr: pages/osx/lpstat.md

# lpstat

> > Printer status information, displays status information about the current classes, jobs, and printers.

Reviewer: This seems a little redundant.
```suggestion
> Display status information about the current classes, jobs, and printers.
```

> More information: <https://ss64.com/osx/lpstat.html>.

> - Show a long listing of printers, classes, or jobs:

Reviewer: Why `or`? Doesn't it print all of those categories?
```suggestion
- Show a long listing of printers, classes, and jobs:
```

`lpstat -l`

> - Force encryption when connecting to the server:

Reviewer: ```suggestion
- Force encryption when connecting to the CUPS server:
```
